builtin/logical/mongodb/path_config_lease.go

```diff
@@ -0,0 +1,107 @@
+package mongodb
+
+import (
+	"fmt"
+	"time"
+
+	"github.com/hashicorp/vault/logical"
+	"github.com/hashicorp/vault/logical/framework"
+)
+
+func pathConfigLease(b *backend) *framework.Path {
+	return &framework.Path{
+		Pattern: "config/lease",
+		Fields: map[string]*framework.FieldSchema{
+			"ttl": &framework.FieldSchema{
+				Type:        framework.TypeString,
+				Description: "Default ttl for credentials.",
+			},
+
+			"ttl_max": &framework.FieldSchema{
```

Comment:
It looks like there's a variety of names in use for this field:

* The `aws`, `mysql`, and `postgresql` backends use `lease_max`
* The `pki` and `rabbitmq` backends use `max_ttl`
* The `mssql` backend uses `ttl_max`

I had chosen `ttl_max` due to its use in the `mssql` backend, which is newer than most others. However, I don't mind changing it to `max_ttl`; I'll do that now.